app/kumactl/data/install/k8s/gateway/kong/kong.yaml

```diff
@@ -0,0 +1,731 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Namespace }}
```

Comment:
We have the namespace twice. Either delete it from here or from the separate file

Reply:
The namespace should be annotated with `kuma.io/sidecar-injection: enabled`. Otherwise, we won't inject Kuma DP.
https://kuma.io/docs/1.1.4/documentation/dps-and-data-model/#kubernetes